test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -165,8 +165,11 @@ TEST_P(nntrainerIniTest, initThreetime_n) {
 
 /// @todo add run test could be added with iniTest flag to control skip
 static nntrainer::IniSection nw_base("model", "Type = NeuralNetwork | "
-                                              "batch_size = 32 | "
-                                              "loss = cross");
+                                              "batch_size = 32");
+
+static nntrainer::IniSection nw_base_cross("model", "Type = NeuralNetwork | "
+                                                    "batch_size = 32 | "
+                                                    "loss = cross");
```

Comment:
Could you add tests that loss is at the last layer explicitly?

Reply:
I will add it as a separate PR soon? The purpose of this patch is to enable existing tests and check its working with implemented code.

Reply:
Alright

Reply:
added #1363. Please check.